.github/workflows/ci.yml

```diff
@@ -48,20 +143,58 @@ jobs:
           sudo apt-get -y update
           sudo apt-get -y install protobuf-compiler libprotobuf-dev
 
+      - name: Run cargo test
+        run: |
+          cargo build --locked --no-default-features --features storage-mem
+          (&>/dev/null ./target/debug/surreal start --log trace --user root --pass root memory &)
+          cargo test --locked --package surrealdb --no-default-features --features protocol-http --test api api_integration::http
+
+  mem-engine:
+    name: Memory engine
+    runs-on: ubuntu-20.04
```

Comment:
Yes but that would be overkill. It only takes `2m 57s` currently so it takes less time than the longest running job. Since the jobs run in parallel anyway, improving it doesn't improve the total running time.